### DISCUSSION TITLE
Default value needed if no data is available

### DISCUSSION THREAD

**Discussion by bengtner** [kalkih/mini-graph-card#827] [Ideas]
I find it a bit annoying wrong value (the last value) is shown if there is no data available when you hoover over a bar representing a null value. I have made a post in Home Assistant community forum about this:

https://community.home-assistant.io/t/mini-graph-card-hovering-shows-no-value/448473

A possibility to define a default value will solve this problem.

**Comment by Johnt5818** [comment]
I would also like to see a Default value defined. it would fix a problem when I am displaying Bluetooth historical data. Right now, if the Bluetooth device is offline, it spikes the graph as if it has a great signal.
Having a Default value for when the data is not there, would allow me to adjust this.

Here is my current config. The Red mark is where the Bluetooth device went offline. 
![image](https://github.com/kalkih/mini-graph-card/assets/30358675/d39e1180-885a-4818-9e8a-97492269103a)
